L4/README.md

```diff
@@ -0,0 +1,11 @@
+# L4 setup
+
+The setup in this folder was created for gotm v5. Running a later version of gotm you will need to update the `gotm.yaml` file. This can be done by following the description below.
+
+Option 1:
+1. Download the `gotm` updater from [here](https://github.com/gotm-model/code/blob/master/scripts/python/update_setup.py)
+2. Run `python <path-to-updater> gotm.yaml --gotm <gotm-executable>`, please note you need to run this script with `python` version 3.
+3. Re-run gotm in L4 setups folder
+
+Option 2:
+1. Run `python update-gotm-yaml.py` in the L4 setups folder. You can add the optional argument `--gotm <gotm-excutable>` which will include all extra comments in the yaml file `python update-gotm-yaml.py --gotm <gotm-excutable>`
```

Comment:
I get the error: 
100% [............................................................................] 9552 / 9552  File "update_setup.py", line 149
    print('- Updating configuration to latest version...', end='', flush=True)
                                                              ^
SyntaxError: invalid syntax

I can't even see the update_setup.py file so don't know what is wrong. Interestingly I don't get this error when running the update_setup.py file directly from the GOTM directory

Reply:
This still doesn't work for me but top option now does (I'm also not running in Conda environment so that may be something to do with it?)

Reply:
Do you get the same error as before?

Reply:
Yes, It still doesn't seem to like that print statement

Reply:
It now works...but currently doesn't change the version of gotm.yaml (and therefore doesn't run). I think you need to go back to your original url of master rather v6.0. that seemed to work for me

Reply:
Worth noting that I just fixed an bug in GOTM's update script - that may fix this specific issue

Reply:
I just clarified my comment above - if using the master branch the script works fine.  It doesn't do anything when pointing to the v6.0 branch

Reply:
I must have pushed the updated yaml file to the repo so it wouldn't do anything. I think it is probably best if we stick to a stable version of gotm since, from what Jorn says below, master could contain some unstable code

Reply:
Yes - I'd suggest going for GOTM's v6.0 branch, which is stable (also its yaml format) and will still get (fully backwards compatible) bug fixes as required.

Reply:
Ok, so to clarify for myself before accepting approving everything: The update script that Michael wrote works for updating the gotm.yaml file to version 6 and so therefore works when using the v6.0 branch of gotm. If using the latest branch of GOTM this doesn't work but we are directing people to build with GOTM v6.0 anyhow.  Is this correct? If yes, I will go ahead and approve .

Reply:
Yes that sounds correct. As the GOTM version changes we can update as needed